docker-compose.yml

```diff
@@ -16,3 +16,4 @@ services:
       - "9615:9615"
       - "9944:9944"
       - "9933:9933"
+    command: [ "substrate-contracts-node", "--dev", "-d", "./chain-data", "--unsafe-ws-external", "--rpc-external", "-lerror,runtime::contracts=debug" ]
```

Comment:
Yes, this is correct. Hopefully you can override the entry point via docker compose like this. If not, I've built a separate image called `prosopo/substrate:v1.0.0` which has the `CMD` in the Dockerfile as follows:

```
CMD exec /bin/bash -c "substrate-contracts-node --dev -d ./chain-data --unsafe-ws-external --rpc-external --prometheus-external -lerror,runtime::contracts=debug"
```

The image is in docker hub.